src/QueryComplexity.ts

```diff
@@ -135,7 +134,6 @@ export default class QueryComplexity {
     }
 
     this.estimators = options.estimators || [
-      legacyEstimator(),
       simpleEstimator()
     ];
```

Comment:
@ivome this might be a good time to make `options.estimators` required, since this PR requires changing the default estimators anyway?

Reply:
Good idea, let's make this required now with all the breaking changes.

Reply:
Done